apps/settings/js/modules/bluetooth/bluetooth_connection_manager.js

```diff
@@ -0,0 +1,674 @@
+/**
+ * BluetoothConnectionManager:
+ *   - BluetoothConnectionManager is an Observable that manage the connection
+ *     state.
+ *   - It has some observable properties: connectingAddress, connectedAddress.
+ * BluetoothConnectionManager only update state and does not involve in any UI
+ *   logic.
+ *
+ * @module BluetoothConnectionManager
+ */
+define(function(require) {
+  'use strict';
+
+  var AdapterManager = require('modules/bluetooth/bluetooth_adapter_manager');
+  var Observable = require('modules/mvvm/observable');
+  require('shared/async_storage');
+
+  var _debug = false;
+  var Debug = function() {};
+  if (_debug) {
+    Debug = function btam_debug(msg) {
+      console.log('--> [BluetoothConnectionManager]: ' + msg);
+    };
+  }
+
+  /**
+   * @alias module:modules/bluetooth/BluetoothConnectionManager
+   * @requires module:modules/mvvm/observable
+   * @return {BluetoothConnectionManager}
+   */
+  var BluetoothConnectionManager = {
+    /**
+     * The profile of connected device that we are defined here.
+     *
+     * @public
+     * @memberOf BluetoothConnectionManager
+     * @type {Object}
+     */
+    Profiles: {
+      'HFP': 0x111E,
+      'A2DP': 0x110D
+    },
+
+    /**
+     * The address of device that we are trying to connect.
+     *
+     * @public
+     * @memberOf BluetoothConnectionManager
+     * @type {String}
+     */
+    connectingAddress: null,
+
+    /**
+     * The address of device that we are connected.
+     *
+     * @public
+     * @memberOf BluetoothConnectionManager
+     * @type {String}
+     */
+    connectedAddress: null,
```

Comment:
The item stored in the array can contain the information of the connected profiles.